_studio/shared/umc/codec/vc1_dec/src/umc_vc1_video_decoder.cpp

```diff
@@ -467,7 +467,7 @@ Status VC1VideoDecoder::StartCodesProcessing(uint8_t*   pBStream,
                 (uint8_t*)m_frameData->GetDataPointer() + *pOffsets,
                 UnitSize);
             //use own buffer
-            SwapData(m_dataBuffer, mfx::align_value<uint32_t>(UnitSize));
+            SwapData(m_dataBuffer, mfx::align2_value<uint32_t>(UnitSize));
```

Comment:
A lot of explicit `uint32_t` here and above, do we really need them?

Reply:
no